docs/gettingstarted/structuring.md

```diff
@@ -1,124 +1,81 @@
-```toml
-# The name of the mod loader type to load - for regular FML @Mod mods it should be javafml
-modLoader="javafml"
-# A version range to match for said mod loader - for regular FML @Mod it will be the forge version
-# Forge for 1.19.4 is version 45
-loaderVersion="[45,)"
-# The license for your mod. This is mandatory and allows for easier comprehension of your redistributive properties.
-# Review your options at https://choosealicense.com/. All rights reserved is the default copyright stance, and is thus the default here.
-license="All Rights Reserved"
-# A URL to refer people to when problems occur with this mod
-issueTrackerURL="github.com/MinecraftForge/MinecraftForge/issues"
-# If the mods defined in this file should show as separate resource packs
-showAsResourcePack=false
-
-[[mods]]
-  modId="examplemod"
-  version="1.0.0.0"
-  displayName="Example Mod"
-  updateJSONURL="minecraftforge.net/versions.json"
-  displayURL="minecraftforge.net"
-  logoFile="logo.png"
-  credits="I'd like to thank my mother and father."
-  authors="Author"
-  description='''
-  Lets you craft dirt into diamonds. This is a traditional mod that has existed for eons. It is ancient. The holy Notch created it. Jeb rainbowfied it. Dinnerbone made it upside down. Etc.
-  '''
-  displayTest="MATCH_VERSION"
-
-  [[dependencies.examplemod]]
-    modId="forge"
-    mandatory=true
-    versionRange="[45,)"
-    ordering="NONE"
-    side="BOTH"
-
-  [[dependencies.examplemod]]
-    modId="minecraft"
-    mandatory=true
-    versionRange="[1.19.4]"
-    ordering="NONE"
-    side="BOTH"
+When structuring your mod, pick a unique, top-level package structure. Many programmers will use the same name for different classes, interfaces, etc. Java allows classes to have the same name as long as they are in different packages. As such, if two classes have the same package with the same name, only one would be loaded, most likely causing the game to crash.
+
+```
+a.jar
+  - com.example.ExampleClass
+b.jar
+  - com.example.ExampleClass // This class will not normally be loaded
 ```
 
-If any string is specified as `${file.jarVersion}`, Forge will replace the string with the **Implementation Version** specified in your jar manifest at runtime. Since the user development environment has no jar manifest to pull from, it will be `NONE` instead. As such, it is usually recommended to leave the `version` field alone. Here is a table of attributes that may be given to a mod, where `mandatory` means there is no default and the absence of the property causes an error.
+This is even more relevant when it comes to loading modules. If there are class files in two packages under the same name in separate modules, this will cause the mod loader to crash on startup since mod modules are exported to the game and other mods.
```

Comment:
Perhaps emphasis should be added on the "to crash on startup" part of the paragraph.
```suggestion
This is even more relevant when it comes to loading modules. If there are class files in two packages under the same name in separate modules, this will cause the mod loader **to crash on startup** since mod modules are exported to the game and other mods.
```